src/Microsoft.Health.Fhir.CosmosDb/Features/Search/FhirCosmosSearchService.cs

```diff
@@ -701,7 +702,7 @@ private async Task<bool> ExecuteSubQueryAsync(List<FhirCosmosResourceWrapper> in
                         break;
                 }
 
-                if (includes.Count >= maxCount)
+                if (includes.Count > maxCount)
```

Comment:
This was previously causing an issue where if there was exactly maxCount items the error would be thrown